fluent-syntax/src/parser.js

```diff
@@ -212,7 +212,7 @@ export default class FluentParser {
       tags = this.getTags(ps);
     }
 
-    if (pattern === undefined && attrs === undefined && tags === undefined) {
```

Comment:
No, you're right! That's a good point. The EBNF confirms this: https://github.com/projectfluent/fluent/blob/master/spec/fluent.ebnf#L43. Thanks for correcting me.